src/types.ts

```diff
@@ -61,7 +61,9 @@ import {
   VolumeRestrictionTMEventArgs,
   VestingEscrowWalletContract,
   VestingEscrowWalletEvents,
-  VestingEscrowWalletUnpauseEventArgs,
```

Comment:
Why aren't you exporting the event args?

Reply:
It was a typo mistake, I added "Unpause" but in types we should only import the generic EventArgs for each wrapper.

Reply:
What I meant was, why isn't `VestingEscrowWalletEventArgs` being exported?